src/dbFirebase.js

```diff
@@ -103,17 +103,19 @@ function onPhotosToModerate(fn) {
   });
 }
 
-async function rejectPhoto(photoId) {
+async function rejectPhoto(photoId,userId) {
   return await firestore.collection('photos').doc(photoId).update({
     moderated: firebase.firestore.FieldValue.serverTimestamp(),
-    published: false
+    published: false,
+    moderator: userId
   });
 }
 
-async function approvePhoto(photoId) {
+async function approvePhoto(photoId,userId) {
   return await firestore.collection('photos').doc(photoId).update({
     moderated: firebase.firestore.FieldValue.serverTimestamp(),
-    published: true
+    published: true,
+    moderator: userId
```

Comment:
as the moderator has a document in firestore, can we save the reference to it ? https://firebase.google.com/docs/firestore/manage-data/data-types

Reply:
I will take care of it